ArchiSteamFarm/IPC/ArchiKestrel.cs

```diff
@@ -379,6 +389,14 @@ internal static class ArchiKestrel {
 					mvc.AddApplicationPart(assembly);
 				}
 			}
+
+			foreach (IWebServiceProvider plugin in PluginsCore.ActivePlugins.OfType<IWebServiceProvider>()) {
```

Comment:
Add `if (PluginsCore.ActivePlugins.Count > 0) {` or remove it from call above so it's unified. Since we're only doing foreach, it doesn't make much difference whether we have if or not.

Reply:
Look 10 lines up. `ActivePlugins.Count` is already checked against null here. I added the `if` to the other call for it to be consistant with this section here.